### PR TITLE
Describe how to enable shell completions

### PR DESCRIPTION
**Pull request by ZhukovGreen**
Shell completions is always a good thing:) Let's share that tmt has it!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -242,6 +242,23 @@ packages on your system::
...
+
+For Zsh, add this to ~/.zshrc::
+
+    eval "$(_TMT_COMPLETE=source_zsh tmt)"
+
+For Fish, add this to ~/.config/fish/completions/tmt.fish::
+
+    eval (env _TMT_COMPLETE=source_fish tmt)
+
+Open a new shell to enable completion. Or run the eval command directly in your current shell to enable it temporarily.
```
Comment (thrix) [inline README.rst:260]:
```suggestion
Open a new shell to enable completion. Or run the ``eval`` command directly in your current shell to enable it temporarily.
```

**Review by thrix — APPROVED** [review]
just a small comment, lgtm otherwise

**Comment by psss** [comment]
Thanks for improving the docs. The shell completion for bash should work out-of-the-box because [/etc/bash_completion.d/tmt](https://github.com/psss/tmt/blob/master/bin/complete) is included in the core `tmt` package. But documenting this could be still useful for those who are using other shells or install using pip. Shall we add completion scripts under `/etc/` for other shells as well?

File: README.rst (hunk 1 above)
Comment (ZhukovGreen) [reply README.rst]: fixed

**Comment by ZhukovGreen** [comment]
In my case, I was using a kind of pip install TMT, because I am usually installing python packages this way (details: pyenv + poetry + POETRY_VIRTUALENVS_CREATE=0). This allows me to get the most recent versions and do not depend on the OS provided python.

I think there is a lot of users of fish and zsh shells. Usually, I see that apps provide a `completion` command. So you might have something like:
```
tmt completions bash
tmt completions zsh
tmt completions fish
```
And then describe how to add those to your shell (i.e. https://kubernetes.io/docs/tasks/tools/install-kubectl/#enabling-shell-autocompletion)

**Review by psss — APPROVED** [review]
Looks good, added just a couple of minor adjustments.

**Comment by psss** [comment]
> Usually, I see that apps provide a `completion` command.

This sounds like a very nice idea. Let's track this under #611.

**Pull request merged**